dns/dyndns/src/www/services_dyndns_edit.php

```diff
@@ -340,11 +341,12 @@ function is_dyndns_username($uname)
                     <td><a id="help_for_password" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?= gettext("Password") ?></td>
                     <td>
                       <input name="password" type="password" id="password" value="<?= $pconfig['password'] ?>" />
-                      <div class="hidden" data-for="help_for_password">
-                        <?=gettext('FreeDNS (freedns.afraid.org): Enter your "Authentication Token" provided by FreeDNS.') ?>
-                        <br /><?= gettext('Route 53: Enter your Secret Access Key.') ?>
-                        <br /><?= gettext('Duck DNS: Leave blank.') ?>
-                      </div>
+                        <div class="hidden" data-for="help_for_password">
```

Comment:
The indent here is off (it was ok before)

Reply:
> The indent here is off (it was ok before)

Thanks, typo here. I did saw, that the indentation was "tab indented", as everything else is "char indented". So i did correct, but made the typo. Please recheck. (Not needed for the patch ....)